.github/workflows/ubuntu_build.yml

```diff
@@ -0,0 +1,17 @@
+# https://github.com/motioneye-project/motioneye/wiki/%28Install-On-Ubuntu-%2820.04-or-Newer%29
+name: ubuntu_build
+on: [pull_request, push]
+jobs:
+  ubuntu_build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

Comment:
```suggestion
      # https://github.com/actions/setup-python/releases
      - uses: actions/setup-python@v3
        with:
          # https://github.com/actions/python-versions/releases
          python-version: '3.9'
      - uses: actions/checkout@v3
```
So we use the GitHub actions Python setup feature instead of installing the APT packages manually. This includes also `pip`, so no need to install it either. And that way we have a defined Python version. Later (or now?) we can also add a version matrix, so the following steps run once per defined Python version. Of course the other APT upgrade/install steps should then run before, so that those are not repeated as well: https://github.com/actions/setup-python#usage

Would need to be tested whether the matrix affects all steps of one job or only the ones after the Python setup step. Otherwise probably it is possible to use a dedicated job for installing/upgrading APT packages and the actual Python steps after that one succeeded on the same host, i.e. with the previously installed/upgraded packages present. But this may be done later, IMO for now it is fine to test with one specific Python version, like the latest patch version of the second latest Python release.

Reply:
Ah sorry, I now realise that this is to explicitly test the steps provided in the Wiki, not the build itself 😅.

Hmm, since this depends on the OS used, I think it cannot be perfectly tested on the "hardcoded" GitHub actions runner. We'd probably require different OS Docker images then to assure that APT lists, builds and environment match the actual OS addressed with the Wiki. But that sounds like overkill.

Reply:
If this is meant to test the instructions given here: https://github.com/motioneye-project/motioneye/wiki/Installation
then in theory it would be required. But I think it is overkill, and then probably sufficient to test it for one OS, i.e. Ubuntu Focal which is currently the base OS for the GitHub actions runners.

Generally it should be fine to test builds with different Python versions we want/need to support with the native setup steps available via GitHub actions and leave most individual OS install tests up to users, as it is not reasonable/impossible to run a CI/CD pipeline for each and every OS + Python combination.

Reply:
We can pytest on lots of OSes and Python versions but most other tools (bandit, black, codespell, mypy, pyupgrade, safety, etc.) will produce identical results on all OSes.

Reply:
I also think that we are able to align different OS install instructions when installing only the Python 3 and in case *-dev package (on Debian-based distros `apt install python3-dev`) and then `pip` and all modules via:
```sh
curl -fO 'https://bootstrap.pypa.io/get-pip.py'
python3 get-pip.py
# Below only until we have a release on PyPI, of course
python3 -m pip install 'https://github.com/motioneye-project/motioneye/archive/dev.zip'
```